can/bus.py

```diff
@@ -243,6 +261,17 @@ def wrapped_stop_method(remove_task=True):
 
         return task
 
+    @staticmethod
+    def _get_messages_sequence(msgs):
+        if not isinstance(msgs, (list, tuple)):
```

Comment:
Can't we check for `typing.Sequence` somehow or simply `collections.Sequence`? Or check for `__iter__` + `__len__`?